include/openssl/cmperr.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Generated by util/mkerr.pl DO NOT EDIT
+ * Copyright 1995-2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef HEADER_CMPERR_H
```

Comment:
this should be OSSL_ prefixed.

Reply:
All the file is auto-generated by `util/mkerr.pl`.
Shall we do this and the change according to your next comment there?

Reply:
We fixed it in util/mkerr.pl, no it generates the OSSL_ prefix too.

Reply:
This general improvement has been moved to #8890.